label_sorter/platforms/ecommerce/amazon.py

```
@@ -47,13 +49,18 @@ def get_page_summary(self) -> dict:
+                    if row['Sl.\nNo'].isnumeric():
                         self.label_items.append(
-                            {"name":row[1], "qty":row[3]}
+                            {"name":row['Description'], "qty":row['Qty']}
                         )
-                
+                return self.label_items
         except AttributeError:
-            raise AttributeError("Check type of the table column")
-        
\ No newline at end of file
+            raise AttributeError("Check type of the table column")
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)